webhook-connector/configure-the-webhook.md

```diff
@@ -4,50 +4,68 @@ description: Configurate the webhook in the Tradecloud portal
 
 # Configure the webhook
 
-## Configure your webhook in the Tradecloud1 portal
+## Configure your webhooks in the Tradecloud1 portal
```

Comment:
Nit; we can remove this header. Now we have 3 header lines saying the same thing..

<img width="861" alt="Screenshot 2023-11-15 at 10 20 27" src="https://github.com/tradecloud/tradecloud-docs-connectors/assets/46396634/8cf7af20-1336-4c71-988e-d7a59ccf1455">

Reply:
First header `Configure the webhook` removed